### ISSUE TITLE
Create initial data for local dev network setup

### ISSUE THREAD

**Issue by vovacha** [playzero/zero-network#73]
- It should work for both standalone and parachain.
- Genesis config is probably not possible to use here for all objects, like proposals and campaigns, due to cross-dependencies between objects.

Implementation details:
1. chain_spec.rs - finalize genesis for the DAOs and treasuries (GameDAO, Game3) creation and token distribution. It's almost done but not pushed to the repo yet.
2. Use some JS script to create a bunch of other DAOs with some members, balances, campaigns, and proposals (finalized and in progress).

**Comment by 2075** [comment]
yes, this could be done with something like a scaffolder script, which we want to create for testing as well.

**Comment by vovacha** [comment]
What's the target repo for this task?

**Comment by vovacha** [comment]
I can think of the following scenario:
1. As a pre-requisite, the script should already have a private key of some faucet accounts to top up child accounts.
2. Create X accounts.
3. Create Y Orgs (different OrgType, AccessModel, FeeModel) using Y accounts from the list.
4. Add Z members to each Org, so one member could be a part of multiple Orgs (not all).
5. Create at least on campaign per each Org.
...
